Chethz/Pacman/README.txt

```diff
@@ -0,0 +1,67 @@
+﻿
+
+PACMAN SIMULATOR
+
+About Simulator
+
+The library simulate a Pacman moving on in a square grid without any obstractions. To start the simulater, Pacman has to be placed on the grid.
```

Comment:
The library **simulates** a Pacman **moving within** a square grid without any **obstructions**